.golangci.yml

```diff
@@ -1,36 +1,150 @@
 linters:
   enable:
+    - bodyclose
+    - deadcode
+    - dogsled
```

Comment:
The only issues `dogsled` caught are in test files. Maybe not necessary?

Reply:
Right now only findings for `dogsled` are on test files, I agree. But it could be a good check for "production" code in the future!

Reply:
After looking it up, I'm sure we want dogsleds enabled. :-)